arch/arc/core/cache.c

```diff
@@ -135,12 +135,44 @@ static void dcache_flush_mlines(u32_t start_addr, u32_t size)
  * @return N/A
```

Comment:
Move this doxygen to the dcache_flush_mlines function

Reply:
This doxygen is already exists for dcache_flush_mlines function. Do you want me to remove this from arch_cache_flush?

Reply:
This is what I'd like to see:
1) The doxygen for the arch functions should always be in the header where the prototype is declared.
2) Any duplicate doxygen at the site of arch function implementations should be removed. https://en.wikipedia.org/wiki/Don%27t_repeat_yourself
3) HOWEVER: any existing comments specific to the implementation of an arch function should be preserved, but not in doxygen format, just a comment.